src/main/java/Gops/Todo.java

```diff
@@ -0,0 +1,23 @@
+package Gops;
+
+public class Todo {
+    protected String itemDescription;
+    protected boolean todoStatus = false;
```

Comment:
Coding standard recommended to name boolean `isDone`. Noticed the same issue elsewhere.